### ISSUE TITLE
Strange jerky animation sometimes on Mac

### ISSUE THREAD

**Issue by ioan-chera** [team-eternity/eternity#151]
Sometimes when I start a game on Mac (I used the front-end), probably into fullscreen SDLGL2D though not surely relevant, interpolation seems to be entirely off: the game looks like it's 'vibrating', as if the linear interpolation calculation is warped off from the correct form. The bug goes away if I quit and restart Eternity, though it may happen consecutive times.

PS: not sure if it's 100% about interpolation or something else, but it does look like something occurring at higher fps than 35.

**Comment by Altazimuth** [comment]
Does this still occur if you turn off vertical mouse aiming?

**Comment by ioan-chera** [comment]
Seems it may not be the Eternity interpolation at fault here. I tested by reducing TICRATE to 5 and sometimes I did see the game running jerkily, but the jerking/zigzagging didn't happen 5 times per second, but with the same frequency as when having TICRATE 35.

What I'm using:

macOS 10.12.3
Screen native resolution 2880x1800
macOS UI scaling 2x
Eternity video mode 1440x900 fullscreen
SDL GL 2D drawing engine

Doesn't matter if I configure the video settings to do dynamic switching between discrete and integrated GPUs.

I may test other cases later (windowed? SDL traditional?)
